butane_core/src/db/sqlite.rs

```diff
@@ -58,7 +58,9 @@ impl SQLiteBackend {
 }
 impl SQLiteBackend {
     fn connect(&self, path: &str) -> Result<SQLiteConnection> {
-        SQLiteConnection::open(Path::new(path))
+        let connection = SQLiteConnection::open(Path::new(path))?;
+        connection.execute("PRAGMA foreign_keys = ON")?;
```

Comment:
I wasnt sure whether this belongs here, or in the `SQLiteConnection::open()`

Reply:
I don't have a real opinion either. They're both part of the same module and this is the only caller so I'm not sure it matters.